cli/cli_handler.cpp

```diff
@@ -127,6 +127,42 @@ void handleTransferESDT(cxxopts::ParseResult const &result)
     std::cerr << "Transaction hash: " << txHash << "\n";
 }
 
+void handleSetNetworkConfig(cxxopts::ParseResult const &result)
```

Comment:
> Do we have a README for the cli interaction, as well (can be added in a future PR)?
Yes, but only shows what the help command do: https://github.com/ElrondNetwork/elrond-sdk-erdcpp/blob/development/examples/examples.md#22-examples-cli. I have a separate task to update it, since new features came along

> I'm not sure whether the CLI is for testing the SDK or it's destined to become a swiss-knife for interacting with the network (similar to erdpy).
No, it's not for testing, just similar to erdpy, but very limited usage.